src/integrator.jl

```diff
@@ -1,9 +1,13 @@
 abstract type AbstractIntegrator end
 
-struct Leapfrog{T<:Real} <: AbstractIntegrator
+struct Leapfrog{T<:AbstractFloat} <: AbstractIntegrator
     ϵ   ::  T
 end
 
+function (::Leapfrog)(ϵ::AbstractFloat)
+    return Leapfrog(ϵ)
```

Comment:
Similarly, add a comment in the code to indicate this will return a new `Leapfrog` with new `ϵ`.

Reply:
Added in a0af7ccc668a2a1e537dc5acdbe8c7bd22f11a86.